pandas/tests/conftest.py

```diff
@@ -0,0 +1,51 @@
+import numpy as np
+import pytest
+
+from pandas import Series
+import pandas.util.testing as tm
+
+_all_indices = [tm.makeBoolIndex(10, name='a'),
+                tm.makeIntIndex(10, name='a'),
+                tm.makeFloatIndex(10, name='a'),
+                tm.makeDateIndex(10, name='a'),
+                tm.makeDateIndex(10, name='a').tz_localize(tz='US/Eastern'),
+                tm.makePeriodIndex(10, name='a'),
+                tm.makeStringIndex(10, name='a'),
+                tm.makeUnicodeIndex(10, name='a')]
+
+_arr = np.random.randn(10)
```

Comment:
what is this for?

Reply:
Thats the array used to create Series objects 

```python
_arr = np.random.randn(10)


_all_series = [Series(_arr, index=tm.makeBoolIndex(10, name='a')),
               Series(_arr, index=tm.makeIntIndex(10, name='a')),
               Series(_arr, index=tm.makeFloatIndex(10, name='a')),
               Series(_arr, index=tm.makeDateIndex(10, name='a')),
               tm.makeDateIndex(10, name='a').tz_localize(
                   tz='US/Eastern').to_series(keep_tz=True),
               Series(_arr, index=tm.makePeriodIndex(10, name='a')),
               Series(_arr, index=tm.makeStringIndex(10, name='a')),
               Series(_arr, index=tm.makeUnicodeIndex(10, name='a'))]
```

Reply:
hmm, let's just make it ``np.arange(10)`` then